### ISSUE TITLE
DRAFT Community Bounty - "Educational Content"

### ISSUE THREAD

**Issue by swasilenko** [Joystream/community-repo#390]
## Educational Content

This bounty was approved by council members and by JSG like official bounty, thread https://testnet.joystream.org/#/proposals/547.

### Description

Goal of this bounty - to fill the Atlas interesting high quality content and attract new users to watch it there and become active users in the future.

Create an educational video where you teach people something: creativity, programming, magic tricks, repairs, etc. you need to tell in the video that you make it for Joystream. Publish your video on Atlas, place link on the video in your social nets and write that you invite people to watch your master-class on Joystream platform. I expect to see workshops, webinars, DIY (diy life hacks).

### Tags

- content creation

#### Rewards
Up to 250$ fo really good content.

Inspector (Bounty Manager) should be appointed by text proposal. He should check new videos for the bounty and every Sunday submit a report on Github and spending proposal for payments to participants. At the beginning all the rewards should be payed to Bounty Manager. Bounty Manager will distribute rewards among participants. 

#### Format

This will be an "open" bounty, where all users can contribute without the need to be assigned.

#### Workflow

1. Bounty Manager should create thread on the forum with all the rules concerning the bounty where participants can upload their works.

2. BM should inform the community about start of the bounty through official discord and telegram. He also should time to time remind the community that bounty is open.

3. BM should check participants' applications. If there are mistakes, he should tell about it to participant and ask to eliminate mistakes.

4. BM should create table with information about every participant and his application (if it's match all the rules) and grade every video based on pre-defined criterias in this table.

5. BM should timely check new applications and every week create a report on Github and spending proposal(s) for payments to participants. Then after approving his proposal(s) he should pay to participants.

#### Grading 

Some examples of grading

Videos are graded on:
- Quality of picture and sound
- Video editing quality
- Length of the video
- Content (interesting to watch, how much is the topic covered)
- Video cover, title and description
- The number of social networks on which you have shared the link to the video

#### Conditions

- Your video need to be original and created for Joystream
- Your video need to be published only on Atlas
- You need to say in the video that you make it for Joystream
- The length of the video should be not less then 7 minutes
- Video and audio should be high quality
- You need to share link on the video in your social nets (instagram, twitter, vk and so on)
- You need to arrange your video accordingly: create a suitable video cover, video title, description under the video

#### Annihilation

- The video or written guides contain anything malicious, or "bad" links
- The video violates any licenses
- Your recording shows anything that violates the ToS or any licenses, even if it's in the background

#### Review period

After the end of 1 week BM will spend up to 72h to verify the videos are in line with the terms. Then, the BM will have a further 72h to grade them. The individual rewards should then be paid within 24h, for a total of 1 week.

#### Status

Draft

**Comment by swasilenko** [comment]
@bwhm hi, please check this draft

**Comment by bwhm** [comment]
Hey,
This is a very good foundation, and I'm happy to use with some small tweaks with your permission.

One thing I'm not sure about is whether it's good to have no restrictions on the category of education. Although a large part of what makes current video services great is the ability to learn just about anything there, it seems not so productive to try and recreate that entirely. Perhaps it's good to at least start with topics that are *somewhat* related to Joystream? 

I get that you are (cleverly) adding the mention Joystream in the video to avoid people taking freely licensed videos from other platforms, but it could still explode.

I suggest adding one or more of the following restrictions:
1. Keep it at least joystream-ish (all things crypto, video, coding, etc)
2. Topic(s) of the week/month, where users can only create education content within some constraints (eg. games, car repair, etc.)
3. Add a weekly/monthly budget.

I would probably opt for 1. only, but I'll let the community decide!

**Comment by swasilenko** [comment]
@bwhm yes, i understand you. to tell the truth, i don't like an idea to create content related to Joystream because i want people to create. this restriction will not let "usual" users to participate in this event. we can ask participant to print Joystream logo and write their discord handle and current date. it will help us to see that they really create their videos for this bounty. what do you think?

**Comment by bwhm** [comment]
@swasilenko I agree that it's more fun if "everyone" can participate, but at the same time, the purpose of the incentives on our testnet isn't (yet at least) to source consumers for the platform. We are intentionally targeting people that can help bootstrap the platform, and manage and maintain it after we're gone. 

Not saying this to belittle people skills, but magic tricks won't really do that. We have often thought about finding some niche content that could make the platform the "goto" place for some particular set of users (twitch for gamers), which would hopefully blossom out to more general/mainstream content, but we haven't found anything we like yet.

For that reason, I'd prefer to make it _somewhat_ related to the platform. That is to say, some skill that will make you an asset. Mind you, that is still a lot of things. To name a few:
- coding
- all things tech
- crypto broadly
- research
- marketing
- video skills (editing, directing, acting, etc.)
- audio skills
- writing/communication skills
- economics
- organizational skills
- etc.

Of course, creating a funny/good video can always draw people in to the community simply through exposure - no doubt about that, but that is to some extent covered by the "Original Content" bounty.

**Comment by mmsaw** [comment]
@bwhm @swasilenko I understand that this may sound unpopular, but nevertheless I believe it is also necessary to make a special emphasis on the young audience, schoolchildren and students, these are those who in 10 years will be active users of the service. In this regard, I believe that the young audience will take video reviews of games or even lessons on some popular cybersport discipline. At one time, it was this kind of content that attracted me to some of the now popular services.

**Comment by swasilenko** [comment]
@mmsaw thanks for your idea. i understand you. but if you read Martin's message again, you will see that now he wants to restrict this bounty and start from something related to the platform. i think in the future we will expand it and add more topics to attract different kinds of users

**Comment by swasilenko** [comment]
@Arseniy2706 this is a good idea! news is important part in crypto world. but it's not an educational content... in some way - yes. but i want people to create some instructions and so on to others who will learn by these videos and improve their skills. we can't improve skills by news. nevertheless, i believe that this is a good idea for another bounty and will attract some "life" in our platform
